### PR TITLE
Fix Documentation Build

### PR DESCRIPTION
**Pull request by awholegunch**
Fix MD5 hashes in build.sh.
Fix documentation for ETL plugins.

[Passes Quick Build.](http://builds.cask.co/browse/CDAP-DOB15-1)
[Passes Checkstyle.](http://builds.cask.co/browse/CDAP-DRC2788-1)

### CODE REVIEW

File: cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst [hunk 1]
```diff
@@ -20,20 +20,9 @@ the logs in a TimePartitionedFileSet.
...
-
-  {
-    "<property name>": "<property value>"
-    ...
-  }
-
-For example, the property names needed for S3 are \"fs.s3n.awsSecretAccessKey\"
-and \"fs.s3n.awsAccessKeyId\".
+**fileSystemProperties:** A JSON string representing a map of properties needed for the
+distributed file system. For example, the property names needed for S3 are
```
Comment (bdmogal) [inline cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst]: this should also be made consistent with the previous?

**Comment by bdmogal** [comment]
One minor comment, rest LGTM

File: cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst]: file.rst : It doesn't have the accessID and accessKey properties, correct? So, unlike the other two, these properties would not be automatically added to the fileSystemProperties?

Comment (bdmogal) [inline cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst]: :+1:

**Pull request merged**